app/controllers/movies_controller.rb

```diff
@@ -46,12 +46,12 @@ def show
 
   def modal
     @list = List.find(params[:list_id]) if params[:list_id].present?
-    @movie = Movie.find_by(tmdb_id: params[:tmdb_id]) || Tmdb::Client.movie(params[:tmdb_id])
+    @movie = Movie.find_by(tmdb_id: params[:tmdb_id]) || Tmdb::Client.get_movie_data(params[:tmdb_id])
```

Comment:
Doobelive these can use `GuaranteedMovie.find_or_initialize_from_api`, but the other PR is now merged, so merging/rebasing should give you answers.